docs/tools-reference/custom-kernels-distros/install-nixos-on-linode/index.md

```diff
@@ -62,42 +66,46 @@ The [NixOS manual](https://nixos.org/nixos/manual/) is the main reference for Ni
 
 ### Prepare the Installer
 
-In your browser, navigate to the [NixOS download page](https://nixos.org/nixos/download.html) and copy the URL from the **Minimal installation CD, 64-bit Intel/AMD** link.
+1.  In your browser, navigate to the [NixOS download page](https://nixos.org/nixos/download.html) and copy the URL from the **Minimal installation CD, 64-bit Intel/AMD** link.
+
+1.  [Boot your Linode into rescue mode](/docs/troubleshooting/rescue-and-rebuild#booting-into-rescue-mode) with the **Installer** disk mounted as `/dev/sda`.
 
-[Boot your Linode into rescue mode](/docs/troubleshooting/rescue-and-rebuild#booting-into-rescue-mode) with the installer disk mounted as `/dev/sda`. Once in rescue mode, run the following command, replacing the URL with the latest 64-bit minimal installation image copied from the [NixOS download page](https://nixos.org/nixos/download.html). This example installs NixOS 19.09:
+1.  Once in rescue mode, click the **Launch Console** link to launch the Finnix rescue console and run the following commands, replacing the URL with the latest 64-bit minimal installation image copied from the [NixOS download page](https://nixos.org/nixos/download.html):
 
-    # Bind the URL you grabbed from the download page to a bash variable
-    iso=<URL for nixos download>
+        # Update SSL certificates to allow HTTPS connections:
+        update-ca-certificates
 
-    # Update SSL certificates to allow HTTPS connections
-    update-ca-certificates
+        # set the iso url to a variable
+        iso=<URL for nixos download>
 
-    # Download the ISO and write it to the installer disk
-    curl $iso | dd of=/dev/sda
+        # Download the ISO and write it to the installer disk:
+        curl -L $iso | dd of=/dev/sda
```

Comment:
This just needed the -L option, so I went back to curl instead of the wget command.